pkg/settings.go

```
@@ -45,6 +45,25 @@ func parseAllowedAuthenticators(raw string) []string {
 }
 
+// consistencyLevels are the levels the driver understands. Anything else is
+// rejected when the config is saved rather than on the first query.
+var consistencyLevels = []string{
+	"ANY", "ONE", "TWO", "THREE", "QUORUM", "ALL",
+	"LOCAL_QUORUM", "EACH_QUORUM", "LOCAL_ONE",
+}
+
+// validateConsistency reports whether the configured consistency level is one
+// the driver accepts.
+func validateConsistency(level string) error {
```

golangci-lint (2.12.2)

[error] func validateConsistency is unused

(unused)